tests/copy_survey_dcip_test.py

```diff
@@ -27,17 +26,13 @@
 from geoh5py.workspace import Workspace
 
 
-def test_copy_survey_dcip():
-
+def test_copy_survey_dcip(tmp_path):
     name = "TestCurrents"
     n_data = 12
+    path = os.path.join(tmp_path, r"testDC.geoh5")
```

Comment:
**suggestion:** use `/` operator since `tmp_path` is `pathlib.Path`
(as per doc https://docs.pytest.org/en/6.2.x/tmpdir.html)
```suggestion
    path = tmp_path / r"testDC.geoh5"
```

Could also add the type for `tmp_path` in the function signature

Reply:
and if `path` has to be used as a string, then simply use `str(path)`

Reply:
suggestions hold for all the tests

Reply:
Done